master/internal/grpcutil/auth.go

```diff
@@ -156,12 +156,16 @@ func GetUser(ctx context.Context) (*model.User, *model.UserSession, error) {
 	var err error
 	userModel, session, err = user.ByToken(ctx, token, &extConfig)
 	if err != nil {
-		if errors.Is(err, sql.ErrNoRows) ||
-			errors.Is(err, db.ErrNotFound) ||
-			errors.Is(err, jwt.ErrTokenExpired) {
+		switch {
+		case errors.Is(err, sql.ErrNoRows):
+			return nil, nil, ErrInvalidCredentials
+		case errors.Is(err, db.ErrNotFound):
 			return nil, nil, ErrInvalidCredentials
+		case errors.Is(err, jwt.ErrTokenExpired):
+			return nil, nil, ErrInvalidCredentials
+		default:
+			return nil, nil, err
```

Comment:
Nit: unless I'm missing a small change here, this looks like zero behavioral change to this code. I'd prefer if these kinds of changes only happened when we were already changing something about the code. When I got here, I read this over and over trying to figure out what I was missing and why this change was in the diff.

Personally, I'm not a big fan of `switch` statements in general (and find the new version of this mildly harder to read than the old one), but that's not as important. I think the `if` version is more readable and natural, but if you really want to use a `switch`, you could do something like this, instead, which I would like better:
```go
switch {
case errors.Is(err, sql.ErrNoRows), errors.Is(err, db.ErrNotFound), errors.Is(err, jwt.ErrTokenExpired):
	return nil, nil, ErrInvalidCredentials
default:
	return nil, nil, err
}
```
or, if you don't like the long line:
```go
switch {
case errors.Is(err, sql.ErrNoRows),
	errors.Is(err, db.ErrNotFound),
	errors.Is(err, jwt.ErrTokenExpired):
	return nil, nil, ErrInvalidCredentials
default:
	return nil, nil, err
}
```
or even
```go
switch {
case errors.Is(err, sql.ErrNoRows):
	fallthrough
case errors.Is(err, db.ErrNotFound):
	fallthrough
case errors.Is(err, jwt.ErrTokenExpired):
	return nil, nil, ErrInvalidCredentials
default:
	return nil, nil, err
}
```

Reply:
Not a problem, I had swapped over to the switch statement w/ enumerated cases so that if we ever decide that we want to respond differently to any of these errors it would be easier to do so, but you've got a fair point! 

I had originally been using `jwt.ErrTokenExpired` as the way to communicate the remote user session had expired, and when I swapped over to a dedicated error, I had forgot to restore this. 

I'm happy to return this to the original if statement!